internal/command/volumes/snapshots/create.go

```diff
@@ -0,0 +1,60 @@
+package snapshots
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/superfly/flyctl/client"
+	"github.com/superfly/flyctl/flaps"
+	"github.com/superfly/flyctl/internal/appconfig"
+	"github.com/superfly/flyctl/internal/command"
+	"github.com/superfly/flyctl/internal/flag"
+)
+
+func newCreate() *cobra.Command {
+	const (
+		short = "Snapshot a volume"
+		long  = "Snapshot a volume\n"
+		usage = "create <volume-id>"
+	)
+
+	cmd := command.New(usage, short, long, create, command.RequireSession)
+
+	cmd.Aliases = []string{"create"}
```

Comment:
no need to alias the command name. It is "extracted" from usage field.

Reply:
TIL! Fixed in the last revision.